### PR TITLE
docs: Add mana api documentation

### PR DESCRIPTION
**Pull request by jkrvivian**
# Description of change

Add mana api documentation

## Type of change
Documentation

## Change checklist
- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by capossele — APPROVED** [review]

File: docs/apis/mana.md [hunk 1]
```diff
@@ -0,0 +1,785 @@
...
+|:-----|:------|:------|
+| `rank`  | int | The rank of a node. |
+| `shortNodeID`  | string | The short ID of a node.   |
+| `nodeID`   | string | The full ID of a node.     |
+| `mana`   | float64 | The amount of consensus mana.     |
+
+
+## `/mana/access/nhighest`
+
+You can get the N highest access mana holders in the network, sorted in descending order.
```
Comment (lzpap) [inline docs/apis/mana.md:355]: maybe we can mention here that if N=0, all nodes that have access mana are returned sorted.

File: docs/apis/mana.md [hunk 2]
```diff
@@ -0,0 +1,785 @@
...
+|field | Type | Description|
+|:-----|:------|:------|
+| `shortNodeID`  | string | The short ID of a node.   |
+| `nodeID`   | string | The full ID of a node.     |
+| `mana`   | float64 | The amount of mana.     |
+
+
+## `/mana/pending`
+
+You can get the mana (bm2) that will be pledged by a given output.
```
Comment (lzpap) [inline docs/apis/mana.md:483]:
```suggestion
Get the amount of base access mana that would be pledged if the given output was spent.
```

File: docs/apis/mana.md [hunk 3]
```diff
@@ -0,0 +1,785 @@
...
+  "mana": 26.5,
+  "outputID": "4a5KkxVfsdFVbf1NBGeGTCjP8Ppsje4YFQg9bu5YGNMSJK1",  
+  "timestamp": 1614924295
+}
+```
+
+### Results
+|Return field | Type | Description|
+|:-----|:------|:------|
+| `mana`   | float64 | The amount of mana to be pledged.     |
```
Comment (lzpap) [inline docs/apis/mana.md:528]:
```suggestion
| `mana`   | float64 | The amount of access base mana to be pledged.     |
```

**Review by lzpap — COMMENTED** [review]
Looks amazing @jkrvivian ! only need small changes

**Pull request merged**